src/utils/types.ts

```diff
@@ -77,6 +77,64 @@ export abstract class APIBase {
     }
 }
 
+export class RAWAPI extends APIBase {
```

Comment:
This has a lot of JSON RPC stuff in it still but we don't really need that for RAWAPI.